projects/storefrontapp/src/app/app.module.ts

```diff
@@ -11,6 +11,7 @@ import { StoreDevtoolsModule } from '@ngrx/store-devtools';
 import { translationChunksConfig, translations } from '@spartacus/assets';
 import { TestConfigModule } from '@spartacus/core';
 import {
+  AsmComponentModule,
```

Comment:
Why import it here?

Reply:
Since ASM is now a CMS component, the way to enable asm is to import the module in the application.  Otherwise and by default, the feature is disabled and absent from the app.

For us to be able to test the feature, I enabled it by default in the development app.

Reply:
I think ASM module should be included in CMSLibModule.

Reply:
Fixed.
As agreed the asm modules are now imported in CMSLibModule for the UI and StorefrontFoundationModule for the core module.